lib/autobuild/import/archive.rb

```diff
@@ -183,6 +183,9 @@ def update_cache(package)
                         if retries = self.retries
                             additional_options << "--tries" << retries
                         end
+			if Autobuild.bsd?
+                            additional_options << "--no-check-certificate"
```

Comment:
???? More info on that ? I definitely don't want to sneak something that critical like this !